README.md

```diff
@@ -23,152 +23,7 @@ Kaito has the following key differentiations compared to most of the mainstream
-  labelSelector:
-    matchLabels:
-      apps: phi-3-5
-inference:
-  preset:
-    name: phi-3.5-mini-instruct
-
-$ kubectl apply -f examples/inference/kaito_workspace_phi_3.5-instruct.yaml
-```
-
-The workspace status can be tracked by running the following command. When the WORKSPACEREADY column becomes `True`, the model has been deployed successfully.
-
-```sh
-$ kubectl get workspace workspace-phi-3-5-mini
-NAME                     INSTANCE                   RESOURCEREADY   INFERENCEREADY   JOBSTARTED   WORKSPACESUCCEEDED   AGE
-workspace-phi-3-5-mini   Standard_NC24ads_A100_v4   True            True                          True                 4h15m
-```
-
-Next, one can find the inference service's cluster ip and use a temporal `curl` pod to test the service endpoint in the cluster.
-
-```sh
-# find service endpoint
-$ kubectl get svc workspace-phi-3-5-mini
-NAME                     TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)            AGE
-workspace-phi-3-5-mini   ClusterIP   <CLUSTERIP>  <none>        80/TCP,29500/TCP   10m
-$ export CLUSTERIP=$(kubectl get svc workspace-phi-3-5-mini -o jsonpath="{.spec.clusterIPs[0]}")
-
-# find available models
-$ kubectl run -it --rm --restart=Never curl --image=curlimages/curl -- curl -s  http://$CLUSTERIP/v1/models | jq
-{
-  "object": "list",
-  "data": [
-    {
-      "id": "phi-3.5-mini-instruct",
-      "object": "model",
-      "created": 1733370094,
-      "owned_by": "vllm",
-      "root": "/workspace/vllm/weights",
-      "parent": null,
-      "max_model_len": 16384
-    }
-  ]
-}
-
-# make an inference call using the model id (phi-3.5-mini-instruct) from previous step
-$ kubectl run -it --rm --restart=Never curl --image=curlimages/curl -- curl -X POST http://$CLUSTERIP/v1/completions \
-  -H "Content-Type: application/json" \
-  -d '{
-    "model": "phi-3.5-mini-instruct",
-    "prompt": "What is kubernetes?",
-    "max_tokens": 7,
-    "temperature": 0
-  }'
-```
-
-## Usage
-
-The detailed usage for Kaito supported models can be found in [**HERE**](presets/README.md). In case users want to deploy their own containerized models, they can provide the pod template in the `inference` field of the workspace custom resource (please see [API definitions](api/v1alpha1/workspace_types.go) for details). The controller will create a deployment workload using all provisioned GPU nodes. Note that currently the controller does **NOT** handle automatic model upgrade. It only creates inference workloads based on the preset configurations if the workloads do not exist.
-
-The number of the supported models in Kaito is growing! Please check [this](./docs/How-to-add-new-models.md) document to see how to add a new supported model.
-
-Starting with version v0.3.0, Kaito supports model fine-tuning and using fine-tuned adapters in the inference service. Refer to the [tuning document](./docs/tuning/README.md) and [inference document](./docs/inference/README.md) for more information.
-
-## FAQ
```

Comment:
Where is this section in the new website?

Reply:
updated